Source/MQTTnet.Server/Internal/MqttClientSessionsManager.cs

```diff
@@ -671,39 +671,39 @@ static bool ShouldPersistSession(MqttConnectedClient connectedClient)
         switch (connectedClient.ChannelAdapter.PacketFormatterAdapter.ProtocolVersion)
         {
             case MqttProtocolVersion.V500:
-            {
-                // MQTT 5.0 section 3.1.2.11.2
-                // The Client and Server MUST store the Session State after the Network Connection is closed if the Session Expiry Interval is greater than 0 [MQTT-3.1.2-23].
-                //
-                // A Client that only wants to process messages while connected will set the Clean Start to 1 and set the Session Expiry Interval to 0.
-                // It will not receive Application Messages published before it connected and has to subscribe afresh to any topics that it is interested
-                // in each time it connects.
-
-                var effectiveSessionExpiryInterval = connectedClient.DisconnectPacket?.SessionExpiryInterval ?? 0U;
-                if (effectiveSessionExpiryInterval == 0U)
                 {
-                    // From RFC: If the Session Expiry Interval is absent, the Session Expiry Interval in the CONNECT packet is used.
-                    effectiveSessionExpiryInterval = connectedClient.ConnectPacket.SessionExpiryInterval;
-                }
+                    // MQTT 5.0 section 3.1.2.11.2
+                    // The Client and Server MUST store the Session State after the Network Connection is closed if the Session Expiry Interval is greater than 0 [MQTT-3.1.2-23].
+                    //
+                    // A Client that only wants to process messages while connected will set the Clean Start to 1 and set the Session Expiry Interval to 0.
+                    // It will not receive Application Messages published before it connected and has to subscribe afresh to any topics that it is interested
+                    // in each time it connects.
+
+                    var effectiveSessionExpiryInterval = connectedClient.DisconnectPacket?.SessionExpiryInterval ?? 0U;
+                    if (effectiveSessionExpiryInterval == 0U)
+                    {
+                        // From RFC: If the Session Expiry Interval is absent, the Session Expiry Interval in the CONNECT packet is used.
+                        effectiveSessionExpiryInterval = connectedClient.ConnectPacket.SessionExpiryInterval;
+                    }
 
-                return effectiveSessionExpiryInterval != 0U;
-            }
+                    return effectiveSessionExpiryInterval != 0U;
+                }
 
             case MqttProtocolVersion.V311:
-            {
-                // MQTT 3.1.1 section 3.1.2.4: persist only if 'not CleanSession'
-                //
-                // If CleanSession is set to 1, the Client and Server MUST discard any previous Session and start a new one.
-                // This Session lasts as long as the Network Connection. State data associated with this Session MUST NOT be
-                // reused in any subsequent Session [MQTT-3.1.2-6].
+                {
```

Comment:
It seems your code style settings are different and thus the switch statements are reformatted without any reason.

Reply:
> It seems your code style settings are different and thus the switch statements are reformatted without any reason.看来您的代码风格设置不同，因此 switch 语句毫无理由地被重新格式化。

Yes, this code formatting style issue bothers me and I don’t know how to deal with it.

Reply:
Do you use Rider or ReSharper? The current style is configured for these tools. If you do not use them, it might be the reason.

Reply:
No, I am only using the community preview version of vs2022, all with default settings.

Reply:
OK then it makes sense that the code style is not applied. A solution may be to migrate to an "editorconfig" file. 
Rider is also free since a few weeks for noncommercial use. Maybe you want to give it a try :smile: 
I will apply the code style to this branch so that we can go ahead.